jira_sync/jira_wrapper.py

```diff
@@ -191,26 +193,67 @@ def assign_to_issue(self, issue: Issue, user: str | None) -> None:
             log.debug("Assigning user %s to %s", user, issue.key)
             self.jira.assign_issue(issue.id, user)
 
-    def add_labels(self, issue: Issue, labels: Collection[str] | str) -> None:
+    def add_labels(self, issue: Issue, labels: Collection[str] | str, changes: dict) -> dict:
         """
         Add label to an issue.
 
         :param issue: Issue object
         :param labels: Label(s) to add
-        """
-        if self.run_mode != JiraRunMode.READ_WRITE:
-            log.info("%s: Skipping adding labels %s to JIRA issue", issue.key, ", ".join(labels))
-            return
+        :param changes: Dictionary containing all the changes for the issue
 
+        :return: Updated dictionary of changes
+        """
         if isinstance(labels, str):
             labels = (labels,)
 
         labels_to_add = [label for label in labels if label not in issue.fields.labels]
 
         if not labels_to_add:
             log.debug("%s: Not adding any labels", issue.key)
-            return
+            return changes
 
         log.debug("%s: Adding labels: %s", issue.key, ", ".join(labels_to_add))
         labels_field_ops = [{"add": label} for label in labels_to_add]
-        issue.update(update={"labels": labels_field_ops})
+        return changes | {"labels": labels_field_ops}
+
+    def add_story_points(self, issue: Issue, story_points: int, changes: dict) -> dict:
+        """
+        Add story points to an issue.
+
+        :param issue: Issue object
+        :param labels: Label(s) to add
```

Comment:
This should describe `story_points`.

Reply:
That was just something I didn't noticed after copy/paste.